modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/exp/agg/Accumulators.java

```diff
@@ -208,6 +210,44 @@
         }
     }
 
+    /** */
+    private static class AnyVal implements Accumulator {
```

Comment:
Does unification AnyVal and SingleVal make sense? Add flag or inheritance to check for SingleVal and shared common code for other.

Reply:
Fixed